card/lib/card/set/event/delayed_event.rb

```diff
@@ -92,29 +71,34 @@ def serialize_value value
     when Time
       { value: value.to_s, type: "time" }
     when Hash
-      {
-        value: value.each_with_object({}) { |(k, v), h| h[k] = serialize_value(v) },
-        type: "hash"
-      }
+      { value: serialize_hash_value(value), type: "hash"}
     when ActionController::Parameters
       serialize_value value.to_unsafe_h
     else
       { value: value }
     end
   end
 
+  def serialize_hash_value
+    value.each_with_object({}) { |(k, v), h| h[k] = serialize_value(v) }
+  end
```

Comment:
no logical change intended.

Reply:
In this case you can change the logic to whatever you want. I don't think it is used anymore. I added a patch for ActiveJob that takes care of that (card/config/initializers/02_patches/active_job.rb)

Reply:
:)